.github/workflows/build.yaml

```diff
@@ -68,23 +68,19 @@ jobs:
       sha: ${{ inputs.sha }}
   wheel-build-cuml:
     secrets: inherit
-    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux-build.yml@cuda-120
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-build.yaml@local-script
     with:
       build_type: ${{ inputs.build_type || 'branch' }}
       branch: ${{ inputs.branch }}
       sha: ${{ inputs.sha }}
       date: ${{ inputs.date }}
-      package-name: cuml
-      package-dir: python
-
+      build_script: "ci/build_wheel.sh"
```

Comment:
Did we not set a default for this? I thought we had originally, but now I don't see it in the workflow.